### ISSUE TITLE
Documentation for widely used libraries

### ISSUE THREAD

**Issue by PerryBhandal** [NASA-Tensegrity-Robotics-Toolkit/NTRTsim#62]
Some brief documentation (Where does it belong? In the wiki, Doxygen or NTRTtutorials?) on how to use the general/widely used libraries would be beneficial. This includes JsonCPP, Google Mock and Google Test.

**Comment by brtietz** [comment]
I'll put a vote in for tutorials, that seems the easiest interface to work with for this sort of thing.  Doxygen documentation can then link to the relavent tutorials page.

**Comment by vsunspiral** [comment]
agreed with Brian, plus it will cause us to actually use the tutorial infrastructure, which is a Good Thing (tm).
